Page/signup/signup.html

```diff
@@ -14,46 +15,67 @@
 <body>
     <header>
         <nav class="nav-container">
-            <div class="nav-logo"><a href="/../index.html">Dope Kicks <i class="fa-solid fa-boot"></i></a></div>
+            <div class="nav-logo"><a class="text-decoration-none" href="/../index.html">Dope Kicks <i class="fa-solid fa-boot"></i></a></div>
             <input type="search" placeholder=" Search items here" class="nav-input">
-            <button class="nav-btn"><a href="/page/login/login.html">Login</a></button>
+            <button class="nav-btn"><a class="text-decoration-none" href="/page/login/login.html">Login</a></button>
             <a class="cart-icon" href="/page/wishlist/wishlist.html"><i class="fa-regular fa-heart"></i></a>
             <a class="cart-icon" href="/page/cart/cart.html"><i class="fa-solid fa-cart-shopping-fast"></i></a>
         </nav>
+    </header>
+
+    <main>
+        <div class="log-container">
+            <div class="log-heading">Sign UP</div>
+            <div class="log-input">
+                <div class="input-group ">
```

Comment:
There is whitespace after a single a class name and ending quote. Will suggest you to use prettier or some other formatter